Document the stricter AENS pointer checks (#477)
aeternity/protocol: AENS.md

The `pointers` can point to one of: account address, oracle id,
channel id or contract id.

From Iris hardfork, the following limitations on pointers apply:

>   - No duplicate keys (a key can only be present once in a list of pointers).

Reviewer: ```suggestion
  - No duplicate keys (a key can appear at most once in a list of pointers).
```
